query_server/query/src/metadata/usage_schema_provider/mod.rs

```diff
@@ -104,12 +104,7 @@ pub fn create_usage_schema_view_table(
         builder.filter(binary_expr(col("tenant"), Operator::Eq, lit(tenant_name)))?
     };
 
-    let cols = builder
-        .schema()
-        .fields()
-        .iter()
-        .filter(|f| f.name().ne("tenant"))
-        .map(|f| col(f.name()));
+    let cols = builder.schema().fields().iter().map(|f| col(f.name()));
 
     let logical_plan = builder.clone().project(cols)?.build()?;
```

Comment:
```suggestion
    let logical_plan = builder.build()?;
```

Projection can be removed directly

Reply:
done